packages/promotion/src/models/promotion-rule-value.ts

```diff
@@ -23,7 +23,7 @@ export default class PromotionRuleValue {
   promotion_rule: PromotionRule
 
   @Property({ columnType: "text" })
-  value: string
+  value: number
```

Comment:
question: Shouldn't this still be kept as a `string`?

Reply:
I think that until we get to a conclusion on the usage of bignumbers, all numbers should be typed as number using the ser/deser wdyt?

Reply:
This is not an amount, though. This is the value of the promotion rule.

E.g.
```
attribute: "customer_group_id",
values: ["group_1", "group_2"],
operator: "in"
```

Isn't it? 🤔

Reply:
You're right, good catch!